modules/apps/expando/expando-web/src/main/java/com/liferay/expando/web/internal/portlet/ExpandoPortlet.java

```diff
@@ -225,6 +245,20 @@ protected void setExpandoColumnService(
 		_expandoColumnService = expandoColumnService;
 	}
 
+	private boolean _checkDefaultLocaleValueInformed(
```

Comment:
@crodriguezy Since both usages of this method are used together with `_getDefaultValue`, could you move the validation logic into that method? This way you don't need to have 2 parts of similar logic in `addExpando` and `UpdateExpando`, and no need to add `MustInformDefaultLocale` either.

Reply:
@crodriguezy By the way, it seems a method which returns `boolean` and doesn't throw exception should be called `isXXXX`.

Reply:
Hey @dantewang thanks for reviewing it!
I have added a new commit following your suggestions. Please let me know if there is anything else to be changed.